src/components/UI/Modal/pages/LoginModal.tsx

```diff
@@ -94,64 +124,42 @@ function createBody(data: ILoginPage): JSX.Element {
 					<span>Would you like to use this configuration?</span>
 				</div>
 			)
-			break
 		case LoginConnectionType.Sso:
-			pageBody = <div>Waiting for you to sign in...</div>
-			break
+			return <div>Waiting for you to sign in...</div>
```

Comment:
Let's make it clearer to the user not to return to this window until he finishes the login process in the browser:
Please go ahead and complete the login process in the opened browser.